src/aind_data_transfer/writers.py

```diff
@@ -38,6 +43,13 @@ def compress_and_write_block(
             experiment_name = read_block["experiment_name"]
             stream_name = read_block["stream_name"]
             zarr_path = output_dir / f"{experiment_name}_{stream_name}.zarr"
+            print(len(str(zarr_path)))
```

Comment:
Can we replace the print statement with a logging.debug message?

Reply:
Oh that's a leftover print. Will remove it!